Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/CorelliPowderCalibrationCreateTest.py

```diff
@@ -30,16 +31,39 @@ def test_exec(self):
                                   RelativeRotation=True)
         MoveInstrumentComponent(Workspace='test_workspace', ComponentName='bank1', X=4.98, y=-0.12, z=0.08,
                                 RelativePosition=False)
+
+        # Both FixSource=True, AdjustSource=True can't be True
+        try:
+            CorelliPowderCalibrationCreate(
+                InputWorkspace='test_workspace', OutputWorkspacesPrefix='cal_', TubeDatabaseDir='/tmp',
+                TofBinning=[300, 1.0, 16666.7], PeakPositions=spacings_reference, FixSource=True, AdjustSource=True,
+                ComponentList='bank1', ComponentMaxTranslation=0.2, ComponentMaxRotation=10)
+        except RuntimeError as error:
+            assert 'Some invalid Properties found' in str(error)
```

Comment:
Might be worth including the specific property in these error message checks, to make the tests more specific

Reply:
As you can see [here](https://github.com/mantidproject/mantid/blob/master/Framework/API/src/Algorithm.cpp#L644) the exception message doesn't referent the failing property so I can't extract that information

Reply:
Understood, sorry, I'd misremembered that output